auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <section id="testimonials-container">
+    <v-container class="quote-container">
+      <p>"We're incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values
+        and the role we want to Play."</p>
+      <p><strong>- BC Benefit Company user</strong></p>
```

Comment:
You might try `&ndash;` or `&hyphen;` and see if it looks any better than `-`.

Reply:
`&ndash;` looks best i think, thanks!